.travis.yml

```diff
@@ -17,6 +18,12 @@ matrix:
       env: TOXENV=django20
     - python: 3.5
       env: TOXENV=django20
+    - python: 2.7
+      env: TOXENV=django21
+    - python: 3.4
+      env: TOXENV=django21
+    - python: 3.5
+      env: TOXENV=django21
```

Comment:
Would you mind adding python 3.6 and 3.7 while you're in here?

Reply:
This is the exclude list. I figured Django 2.0 excludes Python 3.4 and 3.5 to keep the build matrix smaller so I copied that pattern. I'm not sure about the best way to support Python 3.7. There's https://github.com/travis-ci/travis-ci/issues/9815#issuecomment-401756442 but then 3.4 doesn't work according to a subsequent comment.

Reply:
OK, this is an improvement and it's time to deprecate older stuff.

I'd like to run tests on 3.6 at least though, someday... :)

Thank you!

Reply:
Yes, tests run on Python 3.6 (only) for Django 2.0 and 2.1.

Reply:
Ah yeah. Thanks Tim. I wasn't looking in the right places.